src/components/home-card/home-card.js

```diff
@@ -9,7 +9,7 @@ const {data}=useAxios()
     return(
         <div className="bottom-card">
             {data.map(item=>(
-                <Link to='/products' className="bottom-card-links">
+                <Link key={item.id} to='/products' className="bottom-card-links">
                 <div className="card">
                     <div className="card-main-section">
```

Comment:
destructuring can be used for simplification.

Reply:
yes will do it